1/apache/Dockerfile

```diff
@@ -24,5 +24,7 @@ RUN curl -fSL "https://github.com/backdrop/backdrop/archive/refs/tags/${BACKDROP
 # Add custom entrypoint to set BACKDROP_SETTINGS correctly
 COPY docker-entrypoint.sh /entrypoint.sh
 
+EXPOSE 80
```

Comment:
Sorry, let me see if I can explain https://github.com/docker-library/official-images/pull/18171#issuecomment-2568234044 better here -- this image is `FROM php:8.3-apache`, which already includes `EXPOSE 80`, so this image _inherits_ that metadata and does not have to repeat it (same for the FPM variant):

```console
$ crane config backdrop:latest | jq '.config.ExposedPorts'
{
  "80/tcp": {}
}
```

Reply:
Yes thank you and I apologise. This was a miss on my part. I am an integrator and not a backdrop user. I should have inspected the manifest of the base images. Will swiftly correct it.